### PR TITLE
Added the Implementation for Redish cache

### PR DESCRIPTION
**Pull request by karthik-tarento**

### CODE REVIEW

File: src/main/java/org/sunbird/cb/hubservices/serviceimpl/ProfileService.java [hunk 1]
```diff
@@ -64,65 +54,13 @@ public Response multiSearchProfiles(String userId, MultiSearch mSearchRequest, S
 		try {
 			List<String> connectionIdsToExclude = connectionService.findUserConnectionsV2(userId,
 					Constants.Status.APPROVED);
+			List<String> connectionIdsToExcludeForPending = connectionService.findUserConnectionsV2(userId,
+					Constants.Status.PENDING);
```
Comment (karthik-tarento) [inline src/main/java/org/sunbird/cb/hubservices/serviceimpl/ProfileService.java:58]: can we use single neo4j query to find both Approved and Pending ??

File: src/main/java/org/sunbird/cb/hubservices/serviceimpl/UserUtilityService.java [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+    public Map<String, Object> getUserInfoFromRedish(MultiSearch multiSearch, String[] sourceFields, List<String> connectionIdsToExclude) {
+        String departmentName = "";
+        List<String> includeFields = sourceFields != null && !Arrays.asList(sourceFields).isEmpty() ? Arrays.asList(sourceFields) : ProfileUtils.getUserDefaultFields();
+        List<String> tags = new ArrayList<>();
+        Map<String, Object> tagRes = new HashMap<>();
+        for (Search sRequest : multiSearch.getSearch()) {
+            departmentName = (String) sRequest.getValues().get(0);
+            departmentName = departmentName.trim().replaceAll(" ", "");
+            String userInformation = redisCacheMgr.getCache(Constants.USER_LIST + Constants.UNDER_SCORE + departmentName);
+            if (!ObjectUtils.isEmpty(userInformation)) {
```
Comment (karthik-tarento) [inline src/main/java/org/sunbird/cb/hubservices/serviceimpl/UserUtilityService.java]: can we avoid negation ?
just switch the implementation or use isNotEmpty method if available...

File: src/main/java/org/sunbird/cb/hubservices/serviceimpl/ProfileService.java (hunk 1 above)
Comment (Sahil-tarento) [reply src/main/java/org/sunbird/cb/hubservices/serviceimpl/ProfileService.java:58]: I checked no implementation for this

File: src/main/java/org/sunbird/cb/hubservices/serviceimpl/UserUtilityService.java (hunk 2 above)
Comment (Sahil-tarento) [reply src/main/java/org/sunbird/cb/hubservices/serviceimpl/UserUtilityService.java]: Done

**Pull request merged**